0_portfolio_input_check_functions.R

```diff
@@ -254,7 +253,7 @@ map_security_sectors <- function(fin_data, sector_bridge) {
   return(fin_data)
 }
 
-map_comp_sectors <- function(comp_fin_data, sector_bridge) {
+map_comp_sectors <- function(comp_fin_data, sector_bridge, sector_bridge_override) {
```

Comment:
Changes to the signature of a function are risky. I recommend writing some kind of test.